web/src/components/organisms/Settings/Integration/hooks.ts

```diff
@@ -0,0 +1,152 @@
+import { useCallback, useMemo, useState } from "react";
+
+import {
+  Integration,
+  IntegrationMember,
+  Role,
+} from "@reearth-cms/components/molecules/Integration/types";
+import {
+  useGetMeQuery,
+  useAddIntegrationToWorkspaceMutation,
+  Role as GQLRole,
+  useUpdateIntegrationOfWorkspaceMutation,
+} from "@reearth-cms/gql/graphql-client-api";
+
+export default (workspaceId?: string) => {
+  const [selectedConnectionModalIntegration, SetSelectedConnectionModalIntegration] =
+    useState<Integration>();
+  const [selectedIntegrationMember, SetSelectedIntegrationMember] = useState<IntegrationMember>();
+  const [integrationConnectModalShown, setIntegrationConnectModalShown] = useState(false);
+  const [integrationSettingsModalShown, setIntegrationSettingsModalShown] = useState(false);
+  const [searchTerm, setSearchTerm] = useState<string>();
+  const { data, refetch } = useGetMeQuery();
+
+  const workspaces = data?.me?.workspaces;
+  const workspace = workspaces?.find(workspace => workspace.id === workspaceId);
+
+  const fromIntegration = (integration: any) => ({
```

Comment:
`integration: any` should be avoided

Reply:
https://github.com/reearth/reearth-cms/blob/58c8be6313ec45a489b63534dab11973f18ce352/web/src/components/organisms/Settings/Integration/hooks.ts#L28